ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```diff
@@ -9,11 +9,21 @@ oc registry login --to=${SHARED_DIR}/pull-secret-build-farm.json
 echo "Set KUBECONFIG to Hive cluster"
 export KUBECONFIG=/var/run/hypershift-workload-credentials/kubeconfig
 
-AWS_GUEST_INFRA_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
-if [[ ! -f "${AWS_GUEST_INFRA_CREDENTIALS_FILE}" ]]; then
-  echo "AWS credentials file ${AWS_GUEST_INFRA_CREDENTIALS_FILE} not found"
+if [[ "${PLATFORM}" == "aws" ]]; then
+  AWS_GUEST_INFRA_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+  if [[ ! -f "${AWS_GUEST_INFRA_CREDENTIALS_FILE}" ]]; then
+    echo "AWS credentials file ${AWS_GUEST_INFRA_CREDENTIALS_FILE} not found"
+    exit 1
+  fi
+elif [[ "${PLATFORM}" == "powervs" ]]; then
+  export IBMCLOUD_CREDENTIALS="${CLUSTER_PROFILE_DIR}/credentials"
```

Comment:
- need to add a check here for this file before exporting it
- name of the credential file can be more specific to platform and also hide like aws `.powervs`

Reply:
Followed aws approach, added a check and renamed the cress file.